cmd/tracee/cmd/list.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/open-policy-agent/opa/compile"
+	"github.com/spf13/cobra"
+
+	tcmd "github.com/aquasecurity/tracee/pkg/cmd"
+	"github.com/aquasecurity/tracee/pkg/cmd/initialize"
+	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/logger"
+	"github.com/aquasecurity/tracee/pkg/signatures/signature"
+)
+
+func init() {
+	rootCmd.AddCommand(listCmd)
+}
+
+var listCmd = &cobra.Command{
+	Use:     "list",
+	Aliases: []string{"l"},
+	Short:   "List traceable events",
+	Long:    ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		// Get signatures to update event list
+		sigs, err := signature.Find(
+			compile.TargetRego,
+			false,
+			"",
```

Comment:
Not a must, but we can potentially add `viper.GetString("signatures-dir")` here by adding the `signatures-dir` flag

Reply:
Done. Inserted the same `signaturs-dir` root flag to the list command.